pkg/kosmosctl/manifest/manifest_deployments.go

```diff
@@ -29,11 +28,46 @@ spec:
             - --v=4
           resources:
             limits:
-              memory: 500Mi
               cpu: 500m
+              memory: 500Mi
             requests:
               cpu: 500m
               memory: 500Mi
+          readinessProbe:
+            exec:
+              command:
+                - cat
+                - /proc/1/cmdline
+            failureThreshold: 30
+            initialDelaySeconds: 3
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 5
+          livenessProbe:
+            exec:
+              command:
+                - cat
+                - /proc/1/cmdline
+            failureThreshold: 30
+            initialDelaySeconds: 3
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 3
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+      serviceAccountName: clusterlink-network-manager
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: app
+                    operator: In
+                    values:
+                      - clusterlink-network-manager
+              namespaces:
+                - kosmos-system
+              topologyKey: kubernetes.io/hostname
 `
```

Comment:
should update the content to deploy?

Reply:
It needs to be updated. 
Do you want me to modify it together? After canceling the test with privileged: true